pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -1437,6 +1437,305 @@ spec:
 	}
 }
 
+func TestReconcile_ApplyTaskLevelComputeResources(t *testing.T) {
+	multiStepsTask := parse.MustParseTask(t, `
+metadata:
+  name: foo-task
+  namespace: default
+spec:
```

Comment:
same comment as for pipelinerun tests: this test can be simplified

Reply:
yap. will arrange a bit for a meaningful look.